src/components/text-editor/prosemirror-adapter/plugins/link/utils.ts

```diff
@@ -0,0 +1,45 @@
+import { LinkMarkAttrs } from './link-mark-spec';
+
+export function getLinkAttributes(url: string): LinkMarkAttrs {
```

Comment:
It should not be too hard to add some tests for this function :slightly_smiling_face: